ui/api-client/dist/esm/runtime.js

```diff
@@ -285,6 +285,28 @@ export function canConsumeForm(consumes) {
     }
     return false;
 }
+export function camelizeResponseKeys(json) {
```

Comment:
Not sure if this is helpful but we do have a camelizeObjectKeys [util](https://github.com/hashicorp/vault/blob/main/ui/app/utils/camelize-object-keys.js).

Reply:
These don't appear to be a one-to-one match of functionality, but if it's easy enough to amend the existing utils, that might be preferred over having similar duplicate code.

Reply:
This file is actually generated from a mustache template so I had to define the function there. Also the api client will (hopefully soon) live in it's own repo.